docs/en/sql-reference/functions/string-functions.md

```diff
@@ -2637,3 +2637,67 @@ Result:
 1. │ [417784657,728683856,3071092609] │
    └──────────────────────────────────┘
 ```
+
+## stringBytesUniq {#stringbytesuniq}
+
+Counts the number of distinct bytes in a string.
+
+**Syntax**
+
+```sql
+stringBytesUniq(s)
+```
+
+**Arguments**
+
+- `s` — The string to analyze. [String](../data-types/string.md).
+
+**Returned value**
+
+- The number of distinct bytes in the string. [UInt8](../data-types/int-uint.md).
```

Comment:
The number of distinct bytes can range from 0 to 256, while the range of the UInt8 data type is just 0 to 255. Let's change to UInt16.